definitions/json/EnglandWales Scrubbed.json

```diff
@@ -6152,7 +6152,7 @@
   {
     "ID": "sendNotificationResponseTribunal",
     "ListElement": "Yes - view document for details",
-    "ListElementCode": "Yes",
+    "ListElementCode": "Yes - view document for details",
```

Comment:
Has this change been tested against callbacks?

Reply:
yes, tested it locally the value in Citizen Ui changed and Callbacks didnt throw any errors